pkg/controller/job/job_controller.go

```diff
@@ -566,9 +567,39 @@ func (jm *Controller) syncJob(key string) (bool, error) {
 		}
 		if complete {
 			job.Status.Conditions = append(job.Status.Conditions, newCondition(batch.JobComplete, "", ""))
+			jobConditionsChanged = true
 			now := metav1.Now()
 			job.Status.CompletionTime = &now
 			jm.recorder.Event(&job, v1.EventTypeNormal, "Completed", "Job completed")
+		} else if *job.Spec.Suspend {
```

Comment:
we need to update the existing condition if there is one by flipping the status and updating the timestamps. I added the following to the KEP after discussing with reviewers:

"Note that when the suspend flag in the Job spec is flipped from true to false, the Job controller simply updates the existing suspend JobCondition status to false; it does not remove the condition or add a new one."